src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/binary_convolution/binary_convolution_kernel_1x1.cpp

```diff
@@ -93,10 +93,10 @@ JitConstants BinaryConvolutionKernel1x1::GetJitConstants(const binary_convolutio
         jit.AddConstant(MakeJitConstant("FILTER_MASK",
                                         (0xFFFFFFFF >> (ic_pack_size - params.inputs[0].Feature().v % ic_pack_size))));
```

Comment:
Just a question. Is there any possibility that it is due to some casting issue? e.g., the wieght is int8 and the filter mask is 32 bit number macro ...

Reply:
I've dumped cl source code and checked some (src, wei0.s0, INPUT0_TYPE, FILTER_TYPE, FILTER_MASK) types. All types were uint.